devicetypes/smartthings/smartsense-moisture-sensor.src/smartsense-moisture-sensor.groovy

```diff
@@ -118,14 +118,25 @@ private Map parseCatchAllMessage(String description) {
 	if (shouldProcessMessage(cluster)) {
 		switch(cluster.clusterId) {
 			case 0x0001:
-				resultMap = getBatteryResult(cluster.data.last())
+				if (cluster.command == 0x007) {
```

Comment:
This is minor, but commands are 8-bits long and we prefer to write hex values with the appropriate number of digits for the data type so this should be `0x07`. Also the comment should say this is the Configure Reporting Response, not bind msg.